validate.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from datetime import datetime
+import os
+import subprocess
+import sys
+
+from libs.build_params import INTERVENTIONS, OUTPUT_DIR, US_STATE_ABBREV
+
+def run_run_py(datasource: str) -> None:
+  '''
+  Check that we successfully ran `python run.py`
+  '''
+  env = os.environ.copy()
+  env['COVID_DATA_PUBLIC'] = datasource
+  start = datetime.now()
+  # Will raise an appropriate exception if run.py crashed
+  subprocess.run(['python', 'run.py'], cwd=os.getcwd(),
+      #shell=True,
+      env=env, capture_output=True, check=True)
+  end = datetime.now()
+  duration = end - start
+  print(f'run.py duration {duration}')
+
+def clear_result_dir(result_dir: str) -> None:
+  for f in os.listdir(result_dir):
+    os.unlink(os.path.join(result_dir, f))
+
+
+UNSUPPORTED_REGIONS = [
+  'AS',
+  'GU',
+  'MP'
+]
+
+def validate_results(result_dir: str) -> None:
```

Comment:
This is great, I think it's worth figuring out how to run this regularly in pytest as part of the normal tests.  But I think we can do that in another PR

Reply:
Sounds good, will do